tdrs-frontend/nginx/cloud.gov/ip_whitelist_ipv6.conf

```diff
@@ -537256,6 +537256,8 @@ allow 2A07:1500:C1C1:C1C1:C1C1:C100::/89;
 allow 2A07:1500:C1C1:C1C1:C1C1:C180::/90;
 allow 2A07:1500:C1C1:C1C1:C1C1:C1C0::/96;
 allow 2A07:1500:C1C1:C1C1:C1C1:C1C1::/97;
+allow 2a09:bac0:1001:347::2dc:10::/82
+allow 2a09:bac0:1001:347::281:4d::/82
```

Comment:
perhaps missing `;` at the end of the lines?

Reply:
totally right